ubuntudesign/developer.ubuntu.com
# Simplify linking to assets

Remove the `assets` templatetag and replace it with `ASSET_SERVER_URL`, the method we use in out other projects.

This means we'll now be always linking directly to the assets server. We've decided this is the best way to do things, as there is little evidence that linking to `assets.ubuntu.com` from China is currently an issue, and we're hoping to provide a GeoDNS CDN in the long run anyway to make it even more robust.

QA
--

Run the site with `make run`, then visit http://127.0.0.1:8017/en/, check it looks okay. Then view the source, check there are URLs linking to assets.ubuntu.com, and follow a few to check the resolve properly.
